profiles/api/__init__.py

```diff
@@ -0,0 +1 @@
+from .ping import ping_bp
```

Comment:
this allows `from profiles.api import ping_bp` rather than `from profiles.api.pin import ping_bp`, so in the same way can be useful to group all the objects that need to be exposed but it's only an aliasing operation